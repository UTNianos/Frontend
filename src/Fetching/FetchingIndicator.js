import React from 'react';
import { Spin, Alert } from 'antd';

const indicatorStyles = {
  marginLeft: '0%',
  marginTop: '10%'
};

const FetchingIndicator = () => {	
  return(
	<div style={indicatorStyles}>
	   <Spin tip="Cargando...">
       <Alert
          message="Cargando"
          description="Por favor espere..."
          type="info"
       />
       </Spin>
	</div>
  );	
}

export default FetchingIndicator;