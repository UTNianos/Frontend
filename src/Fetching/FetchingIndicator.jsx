import React from 'react';

const FetchingIndicator = () => (
  <div>
    <p>Loading...</p>
  </div>
  );

export default FetchingIndicator;
