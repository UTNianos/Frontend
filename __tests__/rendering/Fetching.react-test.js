'use strict';

import React from 'react';
import Fetching from '../../src/Fetching/FetchingIndicator.jsx';
import renderer from 'react-test-renderer';
 
describe('<Fetching />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Fetching />).toJSON();
    expect(tree).toMatchSnapshot();
  })
});

