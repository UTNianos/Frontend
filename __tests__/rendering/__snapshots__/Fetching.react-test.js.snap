// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Fetching /> renders correctly 1`] = `
<div>
  <p>
    Loading...
  </p>
</div>
`;
